src/App.js

```diff
@@ -1,26 +1,10 @@
-/* eslint-disable quotes */
-import logo from "./logo.svg";
-import "./App.css";
+/* eslint-disable quotes, jsx-quotes */
+import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className='App container '>
+      <Calculator />
     </div>
   );
 }
```

Comment:
- [ ] At this moment, your app does not display as expected on the web browser. Take a look at what I have on my browser. :point_down: 
![your-app](https://user-images.githubusercontent.com/26597432/211645392-d6cb7f1d-89a9-4254-8d9e-5570bf2dc28a.png)

- Expected display on the browser :point_down: 
![expected](https://user-images.githubusercontent.com/26597432/211645627-0116e0d3-4d3d-491e-9a17-9487e926f1b2.png)